frontend/packages/kubevirt-plugin/src/components/connected-vm-console/vm-console-page.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { connectToFlags, FlagsObject } from '@console/internal/reducers/features';
+import ConnectedVMConsole from '@console/kubevirt-plugin/src/components/connected-vm-console/connected-vm-console';
+import { ConsoleType } from '@console/kubevirt-plugin/src/constants/vm/console-type';
+import { FLAG_KUBEVIRT } from '@console/kubevirt-plugin/src/plugin';
+
+const VMConsolePage: React.FC<VMConsolePageProps> = ({ match, location, flags }) => {
+  const params = new URLSearchParams(location.search);
+  const type = ConsoleType.fromString(params.get('type'));
+  const { name, ns: namespace } = match.params;
+
+  return (
+    <ConnectedVMConsole
+      isKubevirt={flags.KUBEVIRT}
```

Comment:
we shouldn't send the `|| VNC` type but null so we don't have to show the alert when it is not available.